patterns/page-cv-bio.php

```diff
@@ -24,10 +24,10 @@
 			<div class="wp-block-column" style="flex-basis:65%">
 				<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
 				<div class="wp-block-group">
-					<!-- wp:paragraph {"align":"left","style":{"typography":{"fontSize":"22rem","letterSpacing":"-0.03em"}}} -->
-					<p class="has-text-align-left" style="font-size:22rem;letter-spacing:-0.03em"><?php echo esc_html_x( 'Hey,', 'Example heading above the content of the CV/Bio pattern.', 'twentytwentyfive' ); ?></p>
-					<!-- /wp:paragraph -->
-
+					<!-- wp:heading {"textAlign":"left","style":{"typography":{"fontSize":"22rem","letterSpacing":"-0.03em","fontStyle":"normal","fontWeight":"300","lineHeight":"1.4"}}} -->
+					<h2 class="wp-block-heading has-text-align-left" style="font-size:22rem;font-style:normal;font-weight:300;letter-spacing:-0.03em;line-height:1.4"><?php echo esc_html_e( 'Hey,', 'twentytwentyfive' ); ?></h2>
```

Comment:
Can we keep the `esc_html_x` here so that there's context for the translators? `Hey,` is a text that can cause some confusion.

Reply:
Changes push in latest commit